### PR TITLE
feat(chat): added emoji picker

### LINKED ISSUE #423: [FEATURE] Enable Users send emojis
### Description

Currently, emojis can only be sent on mobile devices (possible due to emoji keyboard thingy), but not possible on Desktops (unless copied and pasted). It would be great to be able to send emojis in messages by picking the from a menu

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by amitabh-sahu**
# Fixes Issue
**My PR closes #423**

# 👨‍💻 Changes proposed(What did you do ?)
I have added an emoji picker for chat using the emoji-picker-react library.

# ✔️ Check List (Check all the applicable boxes)
- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR are clear and explain the approach.

##  Note to reviewers
This PR has a dependency added on the client side.

# 📷 Screenshots
https://github.com/Dun-sin/Whisper/assets/90052329/8a3395e4-ef45-4c00-ae13-cd5b8925c401

### CODE REVIEW

**Comment by Dun-sin** [comment]
best PR so far🥇
just a change, when the user has picked an emoji, the message box isn't focused back, which is a very annoying inconvenience

File: client/src/components/EmojiPicker.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                </div>
+            </div>
+        </>}
+    </>
+  )
+}
+
+EmojiPicker.propTypes = {
+    onEmojiPick: PropTypes.func,
+};
```
Comment (Dun-sin) [inline client/src/components/EmojiPicker.jsx:49]: add an ending empty line at the end of this file

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by amitabh-sahu** [comment]
Made requested changes

**Review by Dun-sin — APPROVED** [review]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Pull request merged**